compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/DescriptorUtils.kt

```diff
@@ -28,7 +28,7 @@ public fun KotlinType.classDescriptorOrNull(): ClassDescriptor? {
 public fun KotlinType.classDescriptor(): ClassDescriptor {
```

Comment:
Notice the new file name. Before it was `CompilerUtils`, now it's `DescriptorUtils`.